app/uk/gov/hmrc/cdsimportsddsfrontend/domain/PreviousDocument.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cdsimportsddsfrontend.domain
+
+import cats.implicits._
+import scala.xml.Node
+
+case class PreviousDocument(
+                             categoryCode: Option[String] = Some("Y"),
+                             id: Option[String] = Some("20191101"),
+                             typeCode: Option[String] = Some("CLE"),
+                             lineNumeric: Option[String] = Some("lnx") ) {
+
+  def toXml(): Option[Node] = {
```

Comment:
I don't think we should be putting XML rendering code in the domain. The domain should be pure business logic. Let's have a chat about our architecture with all four of us

Reply:
Where do you think it should be?
In the absence of using something like JAXB and not to further pollute DeclarationXML, i'd say this is fine for temporary use.

Reply:
so lets try collect these together in the service layer as implicit conversions

Reply:
Done. Created an XMLWriter.